test/1.4/syntax/T_assign.dml

```diff
@@ -53,4 +53,9 @@ method test() throws {
     i = 3;
     c[++j] += ++i;
     expect(c[1] == 4 && j == 1 && i == 4, "+= operands evaluated twice");
+
+    i = 1;
+    j = 0;
+    (i, j) = (j, i);
+    expect(i == 0 && j == 1, "multiple simultaneous assignment");
```

Comment:
I prefer `assert` over `expect` also in surroundings where `expect` is the dominant function. If you prefer to follow the surrounding pattern then I'm fine with that.